docs/03-tutorials/00-serverless/README.md

```diff
@@ -2,4 +2,70 @@
 title: Serverless - tutorials
 ---
 
+# Overview - Its all about custom resources
+
+Kyma Serverless introduces a [Function](../../05-technical-reference/00-custom-resources/svls-01-function.md) Custom Resource Definition (CRD) as an extension to the Kubernetes API server.
+Defining a Function in Kyma essentially means creating a new instance of the Function custom resource (CR). However, the content of the Function CR specification may become quite long. It consists of the code (or git reference to the code), dependencies, runtime specification, build-time specification, etc. Additionally, there are other CRs that are relevant for a Function developer - i.e. [API Rule](../../05-technical-reference/00-custom-resources/apix-01-apirule.md) (defining how Function is exposed to the outside world), [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) (defining which Cloud Events should trigger Function) and others.
+All of that makes it cumbersome to define the setup by hand. 
+The following sections will guide you through the best practices for the Function development. You will find hints that will be helpful for you at any stage of your development journey.
+
+# Use UI to explore
+
+At the beginning of your Kyma journey you will probably want to evaluate Serverless and draft a few Functions.
+Kyma Dashboard is perfect to gain basic experience and start the journey with Kyma Functions. Dashboard consists of UI components dedicated to Serverless that will help you draft your first Functions by putting the code directly in the browser using Web IDE.
+Kyma Dashboard will also help you expose your Function via HTTP, define environment variables, subscribe to Cloud Events, bind Service Instances and even show you the Function logs - all in single place.
+
+Get started with [Function UI](./svls-01-create-inline-function.md)
+
+![function-ui](./assets/svls-function-ui.png)
+
+# Use Kyma CLI for better development experience
+
+Defining Function from the UI is very quick and easy but most likely this approach would be not enough to satisfy your needs as a developer. The ability to code your Function from your favorite IDE or being able to run and debug the Function on your local machine before actually deploying in the Kyma runtime - Those aspects may be desired especially if you want to code and test a bit more complex cases. Also, you would probably want to avoid recreating same Functions manually from the UI on a different environment. In the end having deployable artefacts is more desirable. This is where Kyma CLI comes in handy as it allows you to keep your Function's code and configuration in a form of a workspace. 
```

Comment:
```suggestion
Defining your Function from the UI is very quick and easy but it might be not enough to satisfy your needs as a developer. The ability to code your Function from your favorite IDE or being able to run and debug the Function on your local machine before actually deploying in the Kyma runtime - Those aspects may be desired especially if you want to code and test more complex cases. Also, you would probably want to avoid recreating the same Functions manually from the UI on a different environment. In the end having deployable artefacts is more desirable. This is where Kyma CLI comes in handy as it allows you to keep your Function's code and configuration in a form of a workspace. 
```